### PR TITLE
chore: Handle Error on getting vote power

### PR DESCRIPTION
**Pull request by Chef-Yogi**
demo

demo2

### CODE REVIEW

File: src/views/Voting/hooks/useGetVotingPower.tsx [hunk 1]
```diff
@@ -27,8 +29,10 @@ const initialState: State = {
 
 const useGetVotingPower = (block?: number, isActive = true): State & { isLoading: boolean } => {
   const { account } = useWeb3React()
+  const { toastError } = useToast()
   const [votingPower, setVotingPower] = useState(initialState)
   const [isLoading, setIsLoading] = useState(!!account)
+  const { t } = useTranslation()
 
   useEffect(() => {
     const fetchVotingPower = async () => {
```
Comment (memoyil) [inline src/views/Voting/hooks/useGetVotingPower.tsx:38]: We can use swr instead

**Comment by memoyil** [comment]
It should not be shown voting power 0 when there is an error, instead of toast error maybe it can be switched to error modal (since there is a visibility timeout on toast it can be missed by user)

**Comment by chefilip** [comment]
fix eslint

File: src/views/Voting/components/CastVoteModal/MainView.tsx [hunk 2]
```diff
@@ -41,8 +42,12 @@ const MainView: React.FC<MainViewProps> = ({
         <Text color="secondary" mb="8px" textTransform="uppercase" fontSize="12px" bold>
           {t('Your Voting Power')}
         </Text>
-        {isLoading ? (
+        {isLoading && !isError ? (
           <Skeleton height="64px" mb="24px" />
+        ) : isError ? (
+          <Text mb="12px" color="red">
+            {t('Error fetching Voting Power')}
```
Comment (chefilip) [inline src/views/Voting/components/CastVoteModal/MainView.tsx:49]: Hmm. The error is a good indicator but it's not informative. As a user, I don't know what the error I'm facing and what the next action I should take to resolve it.

Comment (Chef-Yogi) [reply src/views/Voting/components/CastVoteModal/MainView.tsx:49]: Yeah, useSWR will retry to fetch automatically. (maybe we can tell the user will retry to fetch automatically?)
And about the wording. I am discussing this with Chef Yuna, and she said she wants to reword this message.

**Pull request merged**